bpdm-gate/src/main/resources/db/migration/V4_0_0_5__create_table_legal_entity_identifiers.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE bpdmgate.legal_entity_identifiers (
```

Comment:
Please remove schema names from migration files as this limits the flexibility on where these migration files can be applied to. Flyway handles on which schema this is executed

Reply:
done